### PR TITLE
Add support for PostgreSQL in the main TPC-H files.

### PR DESCRIPTION
**Pull request by DimCitus**
This now has quite some amount of work with no hope to easily sync with TPC-H ever, as this PR contains the full support for the _direct loading_ of the data to PostgreSQL, using the COPY protocol without intermediary files. It might be too invasive for this repository, and we could maybe have that _vendor fork_ of the TPC-H files live elsewhere.

### CODE REVIEW

**Comment by metdos** [comment]
You don't need to use citus_indent here. Also, FYI, we don't use citus_indent on PostgreSQL files like rule_utils which makes integration to newer versions harder.

File: tpch_2_13_0/qgen.c [hunk 1]
```diff
@@ -205,18 +210,23 @@ char *cptr,
                     break;
                 case 'n':
                 case 'N':
+				{
                     if (!(flags & DFLT_NUM))
-                        {
+					{
```
Comment (metdos) [inline tpch_2_13_0/qgen.c]: Such as this change makes it a bit harder to read.

**Comment by metdos** [comment]
I liked the changed output, it would be better if you revert back indentation changes to see modifications better.

**Comment by DimCitus** [comment]
I made more changes and I don't think it's a good idea to publish them here, I'm going to handle that in my own tpch repository with more scripting around it. We can always change our mind later of course.

**Comment by metdos** [comment]
Sounds good!

**Pull request closed without merging**